inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBInstance.java

```diff
@@ -122,17 +122,18 @@ public void write(RepositoryConnection aConn, KnowledgeBase kb)
 
     public static KBInstance read(RepositoryConnection aConn, Statement aStmt)
     {
+        String language = "en";
```

Comment:
Do we have a `language` parameter in the knowledge base object? If not, then I would either add it now or open an issue for it and making a `TODO` here.

Reply:
Open an issue. We need to discuss if a parameter is the right thing to do and what it should be used for.